utils/node_local_test.py

```diff
@@ -3988,6 +3988,76 @@ def test_pydaos_kv(server, conf):
     daos._cleanup()
     log_test(conf, pydaos_log_file.name)
 
+
+def test_pydaos_kv_obj_class(server, conf):
+    """Test the predefined object class works with KV"""
+    # pylint: disable=consider-using-with
+
+    pydaos_log_file = tempfile.NamedTemporaryFile(prefix='kv_objclass_pydaos_',
+                                                  suffix='.log',
+                                                  delete=False)
+
+    os.environ['D_LOG_FILE'] = pydaos_log_file.name
+    daos = import_daos(server, conf)
+
+    pool = server.get_test_pool_obj()
+
+    cont = create_cont(conf, pool, ctype="PYTHON")
+
+    container = daos.DCont(pool.uuid, cont.uuid)
+    failed = False
+    # Write kv1 dictionary with OC_S2 object type
+    kv1 = container.dict('object1', {"Monday": "1"}, "OC_S2")
+    if len(kv1) != 1:
+        failed = True
+        print(f'Expected length of kv object is 1 but got {len(kv1)}')
+
+    # Write kv2 dictionary with OC_SX object type
+    kv2 = container.dict('object2', {"Monday": "1", "Tuesday": "2"}, "OC_SX")
+    if len(kv2) != 2:
+        failed = True
+        print(f'Expected length of kv object is 2 but got {len(kv2)}')
```

Comment:
maybe add 1 more dictionary that does not set the oclass. expected oclass should be S1 in this case.

Reply:
will do